docs/topics/native/native-gradle.md

```diff
@@ -3,117 +3,131 @@
+     distributionType = 'BIN'
+   }
+   ```
 
 </tab>
 </tabs>
 
-Next, create an empty `settings.gradle` or `settings.gradle.kts` file in the project folder.
+   You can use different [target presets](mpp-supported-platforms.md), such as `macosX64`, `mingwX64`, `linuxX64`, `iosX64`,
+   to define corresponding the target platform. The preset name describes a platform for which you are compiling your code.
+   These target presets optionally take the target name as a parameter, which is `native` in this case.
+   The target name is used to generate the source paths and task names in the project.
+
+3. Create an empty `settings.gradle` or `settings.gradle.kts` file in the project folder.
+
+4. Create a folder `src/nativeMain/kotlin` and place inside the `hello.kt` file with the following content:
+   
+   ```kotlin
+   fun main() {
+     println("Hello, Kotlin/Native!")
+   }
+   ```
 
-Depending on the target platform, different [functions](mpp-supported-platforms.md),
-such as `macosX64`, `mingwX64`, `linuxX64`, `iosX64`,
-are used for creating the Kotlin target. The function name is the platform for which you are compiling your code. 
-These functions optionally take the target name as a parameter, which is `"native"` in our case. 
-The specified _target name_ is used to generate the source paths and task names in the project.  
+   By convention, all sources are located in the `src/<target name>[Main|Test]/kotlin` folders, where `main` is for the source code
+   and `test` is for tests. `<target name>` corresponds to the target platform (in this case `native`), as specified in the build file.
 
-By convention, all sources are located in the `src/<target name>[Main|Test]/kotlin` folders, where `main` is for the source code
-and `test` is for tests. `<target name>` corresponds to the target platform (in this case `native`), as specified in the build file. 
+Now you are ready build your project and run the application. 
 
-Create a folder `src/nativeMain/kotlin` and inside it place the file `hello.kt` with the following contents:
+## Build and run the application
 
-```kotlin
-fun main() {
-  println("Hello Kotlin/Native!")
-}
-```
+1. From the root project folder, run the build command:
 
-## Build the project
+   ```bash
+   gradle nativeBinaries
+   ```
 
-From the root project folder, execute the build by running 
+   This command creates a folder `build/bin/native` and two subfolders inside: `debugExecutable` and `releaseExecutable`. They contain corresponding binary files.  
 
-`gradle nativeBinaries`
+   By default, the name of the binary file is the same as the project folder. 
 
-This should create a folder `build/bin/native` with two subfolders `debugExecutable` and `releaseExecutable` with the corresponding binary.
-By default, the binary's name is the same as the project folder. 
+3. To run the project, execute the following command:
+
+   ```bash
+   build/bin/native/debugExecutable/<your_project_folder_name>.kexe
```

Comment:
`your_project_folder_name` seems a bit long and confusing (like: "why the _folder_ name"?). Maybe just `project_name` if it's correct.